frontend/src/modules/search/components/SearchBox.css

```diff
@@ -0,0 +1,84 @@
+.search-box {
+    border-bottom: 1px solid #5e6475;
+    position: static;
+}
+
+.search-box input[type="search"] {
+    background: #4d5967;
+    border: 0;
+    box-sizing: border-box;
+    color: #fff;
+    float: left;
+    font-size: 16px;
+    font-weight: 300;
+    line-height: 23px;
+    padding: 10px 35px 10px 43px;
+    width: 100%;
+
+    /* Remove extra padding in Chrome search input */
```

Comment:
We have other webkit specific styles in `translate.css`. For example, in prod we remove the clear/cancel button in the input field that Chrome shows.

We should also use this opportunity to make the placeholder text in Chrome use the same color as in Firefox.